modules/imaging_browser/README.md

```diff
@@ -0,0 +1,62 @@
+# Imaging Browser
+
+## Purpose
+
+The imaging browser is intended to allow users to view candidate
+scan sessions collected for a study.
+
+It is used primarily by imaging specialists to do online QC of
+imaging data that has been inserted into LORIS.
+
+## Scope
+
+The imaging browser displays images processed images which meet the
+study's inclusion criteria. The inclusion criteria (for most images)
+is defined and enforced in the LORIS imaging pipeline scripts.  Derived
+or processed scan types are also included and have their own insertion
+criteria.
+
+NOT in scope:
+
+The imaging browser module does not display raw DICOMs or perform automated
+quality control on images. It only displays images that have already been
+inserted into LORIS.
+
+## Permissions
+
+The imaging browser module uses the following permissions. Any one of them
+(except imaging_browser_qc) is sufficient to have access to the module.
+
+imaging_browser_view_allsites
+    - This permission gives the user access to all scans in the database
+
+imaging_browser_view_site
+    - This permission gives the user access to scans from their own site only
+
+imaging_browser_phantom_allsites
+    - This permission gives the user access to phantom, but not candidate, scans
+      across the database
+
+imaging_browser_phantom_ownsite
+    - This permission gives the user access to phantom, but not candidate, data
+      only at the user's site.
+
+imaging_browser_qc
+    - This permission gives the user access to modify the quality control data
+      for the associated scans and timepoints.
+
+## Configurations
+
+The imaging browser has the following configurations that affect its usage
+
+tblScanTypes - This setting determines which scan types are considered "NEW" for
+        QC purposes.
+
+useProjects - This setting determines whether "project" filtering dropdowns exist
+        on the menu page.
+
+useEDC - This setting determines whether "EDC" filtering dropdowns exist
+        on the menu page.
+
+mantis_url - This setting defines a URL for LORIS to include a link to for bug reporting
+        on the viewsession page.
```

Comment:
Based on today's SIG meeting, we thought maybe adding a section on interaction with other modules might be appropriate here?

If so, I would add the DQT interaction and how it is expected that the image with the Selected flag is the one that is 'found' by the DQT.

Reply:
I added a section. I included the bit about the selected flag despite the fact that the importer is in the `tools/` directory since it isn't used for anything other than getting the data for this module. (and included lists of where this module links to.)